apps/test/unit/componentLibrary/LinkTest.jsx

```diff
@@ -1,246 +1,85 @@
-//     expect(checkbox).to.exist;
-//
-//     expect(checkbox.checked).to.be.false;
-//     expect(checkbox.disabled).to.be.true;
-//     expect(checkbox.indeterminate).to.be.false;
-//
-//     await user.click(checkbox);
-//
-//     // Re-render after user's first click
-//     rerender(
-//       <Checkbox
-//         name="test-checkbox"
-//         value="test-checkbox"
-//         label="Checkbox label"
-//         checked={checked}
-//         onChange={onChange}
-//         disabled={true}
-//       />
-//     );
-//
-//     checkbox = screen.getByDisplayValue('test-checkbox');
-//
-//     expect(spyOnChange).to.not.have.been.called;
-//     expect(checkbox.checked).to.be.false;
-//     expect(checkbox.disabled).to.be.true;
-//     expect(checkbox.indeterminate).to.be.false;
-//
-//     await user.click(checkbox);
-//
-//     // Re-render after user's second click
-//     rerender(
-//       <Checkbox
-//         name="test-checkbox"
-//         value="test-checkbox"
-//         label="Checkbox label"
-//         checked={checked}
-//         onChange={onChange}
-//         disabled={true}
-//       />
-//     );
-//
-//     expect(spyOnChange).to.not.have.been.called;
-//     expect(checkbox.checked).to.be.false;
-//     expect(checkbox.disabled).to.be.true;
-//     expect(checkbox.indeterminate).to.be.false;
-//   });
-// });
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import sinon from 'sinon';
+
+import {expect} from '../../util/reconfiguredChai';
+
+import Link from '@cdo/apps/componentLibrary/link';
+
+describe('Design System - Link', () => {
+  it('Link - renders with correct text', () => {
+    render(<Link href="https://studio.code.org/home">Home</Link>);
+
+    const link = screen.getByRole('link', {name: 'Home'});
+    expect(link).to.exist;
+    expect(link.href).to.equal('https://studio.code.org/home');
+  });
+
+  it('Link - openInNewTab adds target attribute', () => {
+    render(
+      <Link href="https://studio.code.org/home" openInNewTab>
+        Home
+      </Link>
+    );
+
+    const link = screen.getByRole('link', {name: 'Home'});
+    expect(link).to.exist;
+    expect(link.target).to.equal('_blank');
+    expect(link.href).to.equal('https://studio.code.org/home');
+  });
+
+  it('Link - external adds rel attribute', () => {
+    render(
+      <Link href="https://studio.code.org/home" external>
+        Home
+      </Link>
+    );
+
+    const link = screen.getByRole('link', {name: 'Home'});
+    expect(link).to.exist;
+    expect(link.rel).to.equal('noopener noreferrer');
+    expect(link.href).to.equal('https://studio.code.org/home');
+  });
+
+  it('Link - calls onClick', async () => {
+    const user = userEvent.setup();
+    const spyOnClick = sinon.spy();
+
+    const {rerender} = render(
+      <Link href="/" onClick={spyOnClick}>
+        Home
+      </Link>
```

Comment:
nit: save this component to a local to reuse in `rerender`?

Reply:
Not sure I understand, can you explain or show example maybe please?

Reply:
Oh yep just meant
```
const component =  (
  <Link href="/" onClick={spyOnClick}>
     Home
  </Link>
);

const {rerender} = render(component);
...
rerender(component);
```
Unless they need to be separate components?

Reply:
Oh yeah, nice idea, thanks for spotting this.
I've checked, everything is working correctly. Unless we gonna need to manually update the component's props between `rerender`  - we can use `const component = ...` approach